### PR TITLE
Fix duplication by starts_with and similar_to containing the same package

### LINKED ISSUE #58: The package suggestions seems to be able to show duplicates
Bildschirmfoto 2020-08-19 um 20 37 55

### PR DESCRIPTION
**Pull request by LostKobrakai**
Closes #58 

Would be great if this could get the `hacktoberfest-accepted` label.

### CODE REVIEW

**Review by ericmj — APPROVED** [review]

**Comment by ericmj** [comment]
Thanks for the PR!

> Would be great if this could get the hacktoberfest-accepted label.

What is this?

**Comment by LostKobrakai** [comment]
> What is this?

[Hacktoberfest](https://hacktoberfest.digitalocean.com/) was made opt in this year, so PRs need [explicit approval](https://hacktoberfest.digitalocean.com/hacktoberfest-update) to be counted.

File: test/diff_web/live/search_view_test.exs [hunk 1]
```diff
@@ -66,5 +66,23 @@ defmodule DiffWeb.SearchLiveViewTest do
...
+        "html_sanitize_ex" -> {:ok, ["1.4.0"]}
+        "html_sa" -> {:error, :not_found}
+      end)
+      |> allow(self(), view.pid)
+
+      send(view.pid, {:search, "html_sa"})
+      rendered = render(view)
+      assert rendered =~ "html_sanitize_ex"
+      # one suggestions includes the string twice
+      assert rendered |> :binary.matches("html_sanitize_ex") |> length() == 2
```
Comment (wojtekmach) [inline test/diff_web/live/search_view_test.exs:85]:
```suggestion
      rendered = render(view)
      assert [_] = :binary.matches(rendered, "html_sanitize_ex</span>")
```

**Comment by wojtekmach** [comment]
Thanks!

**Pull request merged**